resources/lang/en/forum.php

```diff
@@ -127,6 +127,49 @@
             'previous' => 'go back 10 posts',
         ],
 
+        'logs' => [
+            '_' => 'Topic logs',
+            'title' => 'topic logs',
+
+            'columns' => [
+                'action' => 'Action',
+                'date' => 'Date',
+                'ip' => 'IP Address',
+                'user' => 'User',
+            ],
+
+            'data' => [
+                'add_tag' => 'added ":data" tag',
+                'announcement' => 'pinned topic and marked as announcement',
+                'delete_post' => 'from topic :data',
+                'edit_topic' => 'to :data',
+                'edit_poll' => 'for :data',
```

Comment:
poll title is in bbcode format so it should be converted accordingly for display

Reply:
since it can be multi-line somehow, think it's better to just not display details for this kinda log entry